README.rst

```diff
@@ -37,19 +37,42 @@ open-mastr
 Introduction
 ============
 
-The `Marktstammdatenregister (MaStR) <https://www.marktstammdatenregister.de/MaStR>`_ is a German register 
-provided by the German Federal Network Agency (Bundesnetzagentur / BNetza) that keeps track of all power and gas units located in Germany.
+The python package ``open-mastr`` provides an interface for accessing the `Marktstammdatenregister (MaStR) <https://www.marktstammdatenregister.de/MaStR>`_ data. The MaStR is a German register provided by the German Federal Network Agency (Bundesnetzagentur / BNetza) that keeps track of all power and gas units located in Germany. It is a daily growing dataset with more than 8.2 million data points covering electricity and gas production units, electricity and gas consumers, storages, grids, and energy market participants (as of spring 2024).
 
-The MaStR data can be
- 
-#. browsed and filtered `online <https://www.marktstammdatenregister.de/MaStR>`_
-#. taken from `daily provided dumps <https://www.marktstammdatenregister.de/MaStR/Datendownload>`_
-#. be accessed via the `web service <https://www.marktstammdatenregister.de/MaStRHilfe/subpages/webdienst.html>`_
+Independent of ``open-mastr``, the MaStR data can generally be accessed via three main options:
```

Comment:
Besides ``open-mastr`` sounds a bit better in my opionion.
Instead of "Independent of ``open-mastr``"

Do you agree?